lib/testutils/httpmultibin/httpmultibin.go

```diff
@@ -165,10 +164,10 @@ func getEncodedHandler(t testing.TB, compressionType httpext.CompressionType) ht
 		)
 
 		switch compressionType {
-		case httpext.CompressionTypeBr:
+		case "br":
 			encw = brotli.NewWriter(rw)
 			encoding = "br"
-		case httpext.CompressionTypeZstd:
+		case "zstd":
```

Comment:
Is this part of the http2 related changes? If not I'd prefer seeing it in a separate commit.

Also, you could just drop the `encoding` var and pass `compressionType` as is (or rename it to `encoding`).

Reply:
this was creating an import cycle so I needed to drop it... I even think it would just be part of the `http` test that does this, but wasn't into trying to tackle the spaghetti that is the `http` module tests